packages/n4s/docs/compound.md

```diff
@@ -129,3 +129,11 @@ enforce({ data: [1, 2, 3] }).shape({
   data: enforce.isArrayOf(enforce.isNumber()),
 });
 ```
+
+## enforce.oneOf()
+
+enforce.oneOf can be used to determine if EXACTLY one of the rules applies. It will run against rule in the array, and will only pass if exactly one rule applies.
```

Comment:
```suggestion
enforce.oneOf can be used to determine if _exactly_ one of the rules applies. It will run against rule in the array, and will only pass if exactly one rule applies.
```